### ISSUE TITLE
Try to recognize a german word

### ISSUE THREAD

**Issue by timaschew** [AASHISHAG/deepspeech-german#6]
Hi, first of all thank you so much for providing this awesome project!

I've installed deepspeach 0.5.0 and run this command:

```
deepspeech --model model-de/model_tuda+voxforge+mozilla.pb --alphabet model-de/alphabet.txt  --lm model-de/lm.binary --trie model-de/trie --audio weihnachtsmann.wav
```

The wav file I've generated from an mp3 which I've got from dict.cc via curl:
```
curl 'https://audio.dict.cc/speak.audio.v2.php?error_as_text=1&type=mp3&id=91424&lang=rec&lp=DEEN' > weihnachtsmann.mp3
```
then converted with ffmpeg via

```
ffmpeg -i weihnachtsmann.mp3 -acodec pcm_s16le -ac 1 -ar 16000  weihnachtsmann.wav
```

and the output is this: `nach man`

I've tried another wave for the same word: 

```
curl 'https://audio.dict.cc/speak.audio.v2.php?error_as_text=1&type=mp3&id=655858&lang=de&lp=DEEN&voice=Hans' > weihnachtsmann.mp3
```

And the result was `reinach zum`.

I was expecting the word is very simple but the result is not really usable. Is the model not trained good enough? I was reading in the paper that the error rate was quite low. So I wondering why this simple word cannot be recognized?

Maybe it is converted with ffmpeg with wrong settings?

**Comment by timaschew** [comment]
Just tried another sentence: **gute nacht allersseits** from

```https://audio.dict.cc/speak.audio.v2.php?error_as_text=1&type=mp3&id=762605&lang=de_rec_ip&lp=DEEN``` 

and it works perfectly 😮

**Comment by zesch** [comment]
Recognition results will vary widely depending on recording conditions, speaker etc.

As the model is currently trained on a relatively small amount of data, it is not very robust when applied to other conditions (see table 5 in the paper for not so nice numbers ;)

We are currently working on improving the model, but the limiting factor is availability of (public) training data.

**Comment by timaschew** [comment]
I was thinking the model was trained on Tuda-De + Voxforge + Mozilla full datasets, no?

>  but the limiting factor is availability of (public) training data.

But Mozill, TuDa-De and Voxforge have public speech corpus', right?
Is it to less data to use it as training data or what is the problem?

**Comment by zesch** [comment]
Even combined, this only amounts to several hundred hours. Commercial models are trained on several thousand hours or even more. This is just not enough for training a really good model. We are currently looking into data augmentation to somewhat lessen that problem.

**Comment by tanujjain** [comment]
Hi

Thanks for the good work. 
Did you try to include ['The Spoken Wikipedia Corpora' ](https://nats.gitlab.io/swc/) for the experiments? It claims to provide more than 300 hours of German data.

On a general note, has there been more progress on the models? (Transfer learning approach/augmentation approaches as you mentioned in an older comment?)

**Comment by AASHISHAG** [comment]
@tanujjain : It's in pipeline. 
We should be able to release the model with new datasets and updated Mozilla version until June.

**Comment by ocastx** [comment]
Is it maybe worth it adding it to the README file as this is a common use case for this project?

**Comment by AASHISHAG** [comment]
@ocastx : Did you mean the command for using the model? 

`deepspeech --model model-de/model_tuda+voxforge+mozilla.pb --alphabet model-de/alphabet.txt  --lm model-de/lm.binary --trie model-de/trie --audio weihnachtsmann.wav`

**Comment by ocastx** [comment]
@AASHISHAG yes, I found that only by searching through DeepSpeech docs which is not intuitive when finding this repo independently. I think this would be very good starting point on getting started with this project. 

If needed, I can do a small proposal PR for this

**Comment by AASHISHAG** [comment]
@ocastx : Yes, please open a PR. I will merge it ASAP.

**Comment by AASHISHAG** [comment]
@tanujjain : We have released v0.6.0. You can find the link in the ReadMe.

**Comment by tanujjain** [comment]
@AASHISHAG Thanks, will check it out.
